src/jail/linux/mod.rs

```diff
@@ -431,18 +437,18 @@ impl LinuxJail {
         )?);
 
         // Write custom resolv.conf that will be bind-mounted into the namespace
-        // Use Google's public DNS servers which are reliable and always accessible
+        // Point to localhost where our dummy DNS server will run
         let resolv_conf_path = format!("/etc/netns/{}/resolv.conf", namespace_name);
         std::fs::write(
             &resolv_conf_path,
             "# Custom DNS for httpjail namespace\n\
-nameserver 8.8.8.8\n\
-nameserver 8.8.4.4\n",
+# Points to local dummy DNS server to prevent exfiltration\n\
```

Comment:
Comment is not exactly right. The proximate reason why we point here is its the only available DNS server and no others would work. It's the only available DNS server because we're trying to prevent exfiltration.